tests/test_utils/dict_compare.py

```diff
@@ -0,0 +1,33 @@
+def compare_dicts(dict1, dict2):
```

Comment:
i created a new package `test_utils` and moved this in here. right now we're using it with two different tests within the cell guide pipeline but it's feasible we'd want to use it in other places as well